crates/katana/storage/db/src/tables.rs

```diff
@@ -242,20 +243,21 @@ mod tests {
         assert_eq!(Tables::ALL[5].name(), TxNumbers::NAME);
         assert_eq!(Tables::ALL[6].name(), TxBlocks::NAME);
         assert_eq!(Tables::ALL[7].name(), TxHashes::NAME);
-        assert_eq!(Tables::ALL[8].name(), Transactions::NAME);
-        assert_eq!(Tables::ALL[9].name(), Receipts::NAME);
-        assert_eq!(Tables::ALL[10].name(), CompiledClassHashes::NAME);
-        assert_eq!(Tables::ALL[11].name(), CompiledClasses::NAME);
-        assert_eq!(Tables::ALL[12].name(), SierraClasses::NAME);
-        assert_eq!(Tables::ALL[13].name(), ContractInfo::NAME);
-        assert_eq!(Tables::ALL[14].name(), ContractStorage::NAME);
-        assert_eq!(Tables::ALL[15].name(), ClassDeclarationBlock::NAME);
-        assert_eq!(Tables::ALL[16].name(), ClassDeclarations::NAME);
-        assert_eq!(Tables::ALL[17].name(), ContractInfoChangeSet::NAME);
-        assert_eq!(Tables::ALL[18].name(), NonceChangeHistory::NAME);
-        assert_eq!(Tables::ALL[19].name(), ClassChangeHistory::NAME);
-        assert_eq!(Tables::ALL[20].name(), StorageChangeHistory::NAME);
-        assert_eq!(Tables::ALL[21].name(), StorageChangeSet::NAME);
+        assert_eq!(Tables::ALL[8].name(), TxExecutions::NAME);
+        assert_eq!(Tables::ALL[9].name(), Transactions::NAME);
+        assert_eq!(Tables::ALL[10].name(), Receipts::NAME);
+        assert_eq!(Tables::ALL[11].name(), CompiledClassHashes::NAME);
+        assert_eq!(Tables::ALL[12].name(), CompiledClasses::NAME);
+        assert_eq!(Tables::ALL[13].name(), SierraClasses::NAME);
+        assert_eq!(Tables::ALL[14].name(), ContractInfo::NAME);
+        assert_eq!(Tables::ALL[15].name(), ContractStorage::NAME);
+        assert_eq!(Tables::ALL[16].name(), ClassDeclarationBlock::NAME);
+        assert_eq!(Tables::ALL[17].name(), ClassDeclarations::NAME);
+        assert_eq!(Tables::ALL[18].name(), ContractInfoChangeSet::NAME);
+        assert_eq!(Tables::ALL[19].name(), NonceChangeHistory::NAME);
+        assert_eq!(Tables::ALL[20].name(), ClassChangeHistory::NAME);
+        assert_eq!(Tables::ALL[21].name(), StorageChangeHistory::NAME);
+        assert_eq!(Tables::ALL[22].name(), StorageChangeSet::NAME);
```

Comment:
Hum.. @kariy I was trying to respect some logical order instead of adding at the end. Do you prefer adding is at the end to avoid big diffs, event if the table is not logically grouped (to keep the incremental offset)?